EvidenceApi/Dockerfile

```diff
@@ -3,11 +3,13 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 WORKDIR /app
 
 # Copy csproj and restore as distinct layers
-COPY *.csproj ./
+COPY ./EvidenceApi/*.csproj ./
 RUN dotnet restore
 
+
 # Copy everything else and build
-COPY . ./
+COPY ./EvidenceApi ./
```

Comment:
Changed this because the Test project was being pulled into the container as well and messing up the file path for the Node module. (Was trying to find /V1/Node -- correctly -- but the container has a directory that included /DocumentsAPI and /DocumentsAPI.Tests, which confused it).